src/lib/cli.ts

```diff
@@ -61,6 +69,23 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
```

Comment:
It looks like there is a package `@types/update-notifier`.
Anyway, I think that all types should go to the folder `/types`